.travis.yml

```diff
@@ -1,58 +1,106 @@
 matrix:
   include:
-    - os: linux
-      dist: trusty
-      sudo: required
-      compiler: gcc
-      addons:
-        apt:
-          sources:
-          - george-edison55/cmake-3.x
-          - ubuntu-toolchain-r-test
-          packages:
-          - cmake
-          - gcc-7
-          - g++-7
-          - autoconf
-          - automake
-          - libtool
-          - curl
-          - make
-          - unzip
-          - libssl-dev
-          - libzmq3-dev
-          - libx11-dev
-          - libxmu-dev
-          - libxss-dev
-      env:
-        - MATRIX_EVAL="CC=gcc-7 && CXX=g++-7"
-
-    - os: osx
-      compiler: clang
-      env:
-        - MATRIX_EVAL="brew install openssl@1.1 && brew install zmq"
-
+  - os: linux
+    dist: trusty
+    sudo: required
+    compiler: gcc
+    before_deploy:
+    - git pull --tags
```

Comment:
`git submodule update --recursive --remote` to have the up-to-date envelopes

Reply:
No, at this stage the binaries are already built. the pull is there only for the purpose of conditionally creating a new tag.